### PR TITLE
Makefile: Fix the issue when $(PLATFORM_PATH) is empty

### PR DESCRIPTION
**Pull request by stcheng**

### CODE REVIEW

File: build_debian.sh [hunk 1]
```diff
@@ -166,6 +166,7 @@ sudo LANG=C chroot $FILESYSTEM_ROOT apt-get -y install      \
     file                    \
     ifupdown                \
     iproute2                \
+    iptables-persistent     \
```
Comment (lguohan) [inline build_debian.sh]: what is this? not related to the commit?

File: slave.mk [hunk 2]
```diff
@@ -51,7 +51,7 @@ distclean : .platform clean
 include $(RULES_PATH)/config
 include $(RULES_PATH)/functions
 include $(RULES_PATH)/*.mk
-ifneq ($(CONFIGURED_PLATFORM), undefined)
+ifneq ($(wildcard $(PLATFORM_PATH)),)
```
Comment (lguohan) [inline slave.mk]: why change this?

File: build_debian.sh (hunk 1 above)
Comment (stcheng) [inline build_debian.sh]: oh this is not related to this pull request.

File: slave.mk (hunk 2 above)
Comment (stcheng) [inline slave.mk]: if the .platform file exists but empty, the make command will fail as the $(CONFIGURED_PLATFORM) is not undefined. With this fix, the make command will not fail unexpectedly.
